interfaces/builtin/udisks2.go

```diff
@@ -375,51 +378,69 @@ func (iface *udisks2Interface) Name() string {
 func (iface *udisks2Interface) StaticInfo() interfaces.StaticInfo {
 	return interfaces.StaticInfo{
 		Summary:              udisks2Summary,
+		ImplicitOnClassic:    true,
 		BaseDeclarationSlots: udisks2BaseDeclarationSlots,
 	}
 }
 
 func (iface *udisks2Interface) DBusConnectedPlug(spec *dbus.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
-	spec.AddSnippet(udisks2ConnectedPlugDBus)
+	if !release.OnClassic {
+		spec.AddSnippet(udisks2ConnectedPlugDBus)
+	}
 	return nil
 }
 
 func (iface *udisks2Interface) DBusPermanentSlot(spec *dbus.Specification, slot *snap.SlotInfo) error {
-	spec.AddSnippet(udisks2PermanentSlotDBus)
+	if !release.OnClassic {
```

Comment:
This is fine, we don't want all the dbus rules for being the udisks service since those are provided by the native udisks package.

Reply:
Exactly.